tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSqsTests.cs

```diff
@@ -54,7 +60,7 @@ public async Task SubmitsTraces(string packageVersion, string metadataSchemaVers
                 var frameworkName = "NetCore";
 #endif
                 var spans = agent.WaitForSpans(expectedCount);
-                var sqsSpans = spans.Where(span => span.Name == "sqs.request");
+                var sqsSpans = spans.Where(span => span.Service == "SQS");
```

Comment:
I hate to be the bearer of bad news, but I think this changed line is causing the `sqsSpans` variable to have 0 elements. This doesn't affect the snapshot tests (those are running against the `spans` variable), but it means we're not testing the SQS spans against our span metadata rules. In order to run against the span metadata rules, which I believe to be a hard requirement for merging this change, we'll need to do the following test updates:

1. Update the clause on this line so it captures the SQS spans, which I think my suggestion below will do. (If you're able to run things locally, you could also temporarily assert that the count of `sqsSpans` is non-zero)
2. Update the file `tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs` so each type of SQS span has the appropriate tag requirements for "peer.service" and "_dd.peer.service.source"

```suggestion
                var sqsSpans = spans.Where(span => span.Tags["component"] == "aws-sdk");
```

Reply:
Sounds about right! Applying both changes.

I wonder why the "temporarily assert", don't we expect them to not be 0 at the end of the day?

Reply:
> I wonder why the "temporarily assert", don't we expect them to not be 0 at the end of the day?

That's a good point, keeping this in the code would be a good safeguard for future changes!